src/core.lisp

```diff
@@ -30,23 +30,19 @@
             (project-point-to-camera camera point))
           triangle))
 
+(defun cross-product-2d (a b)
```

Comment:
I think it is less error-prone to use v:vc from vectors-3d. Just extend vectors by zeros and take the z coordinate.

Reply:
Ok, I'll open an issue then.